ctlearn/build_irf.py

```diff
@@ -428,21 +586,14 @@ def main():
         "_ONLY_THETA": gammas["selected_theta"],
     }
 
-    # binnings for the irfs
-    true_energy_bins = add_overflow_bins(
-        create_bins_per_decade(MIN_ENERGY, MAX_ENERGY, 10)
-    )
-    reco_energy_bins = add_overflow_bins(
-        create_bins_per_decade(MIN_ENERGY, MAX_ENERGY, 5)
-    )
     fov_offset_bins = [0, 0.5] * u.deg
     source_offset_bins = np.arange(0, 1 + 1e-4, 1e-3) * u.deg
     energy_migration_bins = np.geomspace(0.2, 5, 200)
 
     for label, mask in masks.items():
         effective_area = effective_area_per_energy(
             gammas[mask],
-            particles[0]["simulation_info"],
+            particles[1]["simulation_info"],
```

Comment:
While testing your code I find that this change is wrong. Can you please revert to "0"?